clients/include/rocblas.hpp

```diff
@@ -155,26 +151,29 @@
         rocblas_operation transA, rocblas_diagonal diag,
         rocblas_int m, rocblas_int n,
         const T* alpha,
-        const T* A, rocblas_int lda,
-        T*       B, rocblas_int ldb);
+        T* A, rocblas_int lda,
```

Comment:
You can probably keep the const qualifier in the API, but then remove the const with a cast before you call into the kernel.